core/src/main/java/com/exadel/aem/backpack/core/model/OpenPromptModel.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exadel.aem.backpack.core.model;
+
+import com.google.gson.Gson;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Default;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.Required;
+import org.apache.sling.models.annotations.injectorspecific.Self;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+/**
+ * Represents the open prompt props.
+ */
+@Model(adaptables = Resource.class, defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
+public class OpenPromptModel {
+
+    private String name = "backpack.prompt.open";
+
+    @Self
+    @Required
+    private transient Resource resource;
+
+    @Inject
+    @Default(values = "")
+    private String open;
+
+    @Inject
+    @Default(values = "")
+    private String redirect;
+
+    @Inject
+    @Default(values = "")
+    @Named("jcr:title")
+    private String title;
+
+    @Inject
+    @Default(values = "")
```

Comment:
Maybe it's better to use StringUtils.EMPTY

Reply:
done